solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -528,6 +561,57 @@ contract RandomBeacon is IRandomBeacon, Ownable {
         );
     }
 
+    /// @notice Updates the DKG result submission gas.
+    /// @dev Can be called only by the contract owner, which should be the
+    ///      wallet registry governance contract. The caller is responsible for
+    ///      validating parameters.
+    /// @param _dkgResultSubmissionGas DKG result submission gas
+    function updateDkgResultSubmissionGas(uint256 _dkgResultSubmissionGas)
+        external
+        onlyOwner
+    {
+        dkgResultSubmissionGas = _dkgResultSubmissionGas;
+        emit DkgResultSubmissionGasUpdated(_dkgResultSubmissionGas);
+    }
+
+    /// @notice Updates the given gas offsets.
+    /// @dev Can be called only by the contract owner, which should be the
+    ///      wallet registry governance contract. The caller is responsible for
+    ///      validating parameters.
+    /// @param _dkgApprovalGasOffset New DKG approval gas offset
+    /// @param _failedHeartbeatGasOffset New failed heartbeat gas offset
+    /// @param _relayEntrySubmissionGasOffset New relay entry submission gas offset
+    function updateGasOffsets(
+        uint256 _dkgApprovalGasOffset,
+        uint256 _failedHeartbeatGasOffset,
+        uint256 _relayEntrySubmissionGasOffset
+    ) external onlyOwner {
+        dkgApprovalGasOffset = _dkgApprovalGasOffset;
+        failedHeartbeatGasOffset = _failedHeartbeatGasOffset;
+        relayEntrySubmissionGasOffset = _relayEntrySubmissionGasOffset;
+        emit GasOffsetsUpdated(
+            _dkgApprovalGasOffset,
+            _failedHeartbeatGasOffset,
+            _relayEntrySubmissionGasOffset
+        );
+    }
+
+    /// @notice Authorize contract which can request a relay entry without
+    ///         paying fees. Authorization can be done by the owner only.
+    function authorizeContract(address _contract) external onlyOwner {
+        authorizedContracts[_contract] = true;
+
+        emit AuthorizedContract(_contract);
+    }
+
+    /// @notice Unauthorize contract which can request a relay entry without
+    ///         paying fees. Unauthorization can be done by the owner only.
+    function unauthorizeContract(address _contract) external onlyOwner {
+        delete authorizedContracts[_contract];
+
+        emit UnauthorizedContract(_contract);
+    }
```

Comment:
The current approach is *clearer* but if we need to save some bytecode, I would consider doing a similar trick as we did in tBTC v2 contract where we have [a function with a boolean parameter flipping the `isVaultTrusted` boolean flag](https://github.com/keep-network/tbtc-v2/blob/22c72cf5ede18da64ead99741c6cf8b544d4715a/solidity/contracts/bridge/Bridge.sol#L500-L503). This would be something like `setRequesterStatus(address requester, boolean isTrusted)`.

Reply:
Sure, done in [9984f7a](https://github.com/keep-network/keep-core/pull/2860/commits/9984f7abdedd261570a5c6958299dc51c9807ffd)